MAUI/PDF-Viewer/Built-In-Toolbar.md

```diff
@@ -0,0 +1,741 @@
+---
+layout: post
+title: Working with the Built-In Toolbar in .NET MAUI Pdf Viewer (SfPdfViewer) | Syncfusion
+description: Learn here all about the built-in toolbar in .NET MAUI PDF Viewer (SfPdfViewer) control, its functionalites and more.
+platform: MAUI
+control: SfPdfViewer
+documentation: ug
+keywords : .net maui pdf viewer, .net maui view pdf, pdf viewer in .net maui, .net maui open pdf, maui pdf viewer, maui pdf view
+---
+
+# Working with the Built-In Toolbar in .NET MAUI Pdf Viewer (SfPdfViewer)
+
+The [SfPdfViewer](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.PdfViewer.SfPdfViewer.html) has a built-in toolbar, which allows you to perform operations such as adding, modifying annotations, searching text, etc. Additionally, you have the option to show or hide the default built-in toolbar and its sub-toolbars and toolbar items, such as buttons, and you can customize your own toolbar by disabling the built-in toolbar and implementing one at the application and hooking the custom toolbar UI to the PDF viewer APIs. You can see the custom toolbar example [here](https://github.com/SyncfusionExamples/maui-pdf-viewer-examples/tree/master/Custom%20Toolbar).
+
+## Show/hide the built-in toolbar
+
+By default, the built-in toolbar is visible. You can `hide` it by setting the `ShowToolbars` property of the PDF Viewer to `false`. And you can `show` the built-in toolbar by setting the `ShowToolbars` property of the PDF Viewer to `true`.
+
+{% tabs %}
+{% highlight C# %}
+//set the ShowToolbars to false
+public MainPage()
+{
+     SfPdfViewer pdfViewer=new SfPdfViewer();
+     pdfViewer.ShowToolbars = false;
+}  
+{% endhighlight %}
+{% endtabs %}
+
+{% tabs %}
+{% highlight XAML %}
+ <ContentPage.Content>
+
+  <syncfusion:SfPdfViewer x:Name="pdfViewer" DocumentSource ="{Binding PdfDocumentStream} " ShowToolbars="False" />
+
+  </ContentPage.Content>
+{% endhighlight %}
+{% endtabs %}
+
+## Customize the Toolbar 
+
+To customize the toolbar, you can programmatically add, remove, and hide the items based on your needs.
+
+### Customizing toolbar items using the GetByName method
+
+### Accessing and Modifying the Visibility of the Toolbar Items
+
+By using the `GetByName` method with its specified toolbar or icon names, you can access the properties of those toolbars or items. After accessing the item, you can modify its visibility by setting the `IsVisible` property. Here, we retrieve the toolbar "PrimaryToolbar" and "Search" item using the GetByName method and hide the "Search" item by setting its visibility to false.
+
+{% tabs %}
+{% highlight C# %}
+// On the desktop, we access the search button located in the primary toolbar using the GetByName method.
+
+var item = pdfViewer.Toolbars?.GetByName("PrimaryToolbar")?.Items?.GetByName("Search");
+if (item != null)
+{
+item.IsVisible = false; // Hide the search item
+}
+{% endhighlight %}
+{% endtabs %}
+
+### Adding a New Toolbar Item
```

Comment:
**### Adding a New Toolbar Item**

In contents header, only the first letter of the first word should be in **caps**, others are need to be in the small casing.

**Ensure address this in all the areas**

<img width="789" alt="image" src="https://github.com/syncfusion-content/maui-docs/assets/65169077/5bc19e88-f1e8-4c00-8969-c93918239339">